tsconfig.json

```diff
@@ -15,7 +15,12 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "types": ["node", "jest", "webdriverio/async"]
+    "types": ["node", "jest", "webdriverio/async"],
+    "baseUrl": "./",
+    "paths": {
+      "@generated/*": ["src/frontend/generated/*"],
```

Comment:
can you use `$` instead of `@` for prefix?

- org scoped library published to npm are prefixed with `@`
- consistency with NNS-dapp